test/ambiguous.jl

```diff
@@ -141,4 +144,11 @@ let ms = methods(amb_4).ms
     @test Base.isambiguous(ms[3], ms[4])
 end
 
+g{T<:Number}(x::T, y::Integer) = 0
```

Comment:
number this or `make testall1` fails with `ERROR: LoadError: LoadError: cannot define function g; it already has a value`